lib/Demangling/OldRemangler.cpp

```diff
@@ -1267,6 +1267,18 @@ void Remangler::mangleImplResult(Node *node) {
   mangleChildNodes(node); // impl convention, type
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void Remangler::mangleImplDifferentiable(Node *node) {
+  // The old mangler does not encode `@differentiable` function types.
+  Buffer << 'd';
+}
+
+// SWIFT_ENABLE_TENSORFLOW
```

Comment:
This should be lowered ~5 lines, and is missing an ` END`.

Reply:
> This should be lowered ~5 lines,

Are you optimizing the location of these code blocks? I put the code blocks in the order that nodes are defined in `include/swift/Demangling/DemangleNodes.def`.

We haven't been religiously adding `SWIFT_ENABLE_TENSORFLOW END` in all cases, only when the end of the newly added code is ambiguous. I suppose we could start doing so - that requires some team communication.

Reply:
> This should be lowered ~5 lines

Oh, I understand now. You would like to group the entire code block within `SWIFT_ENABLE_TENSORFLOW` and `SWIFT_ENABLE_TENSORFLOW END`. That would be a better organization of these comments, but we haven't been following strict rules for these comments.

Reply:
Optimized comment in https://github.com/apple/swift/pull/26600.